airbyte-integrations/connectors/source-linkedin-leads/source_linkedin_leads/schemas/lead_form.json

```diff
@@ -0,0 +1,284 @@
+{
```

Comment:
Don't do separate json schemas, let's put them in the manifest.

Reply:
why do you prefer this way of working with inline schemas ? IMHO, it is more readable to split them in distinct files and not overload the manifest 🤔